privilege/privilege.go

```diff
@@ -58,10 +58,26 @@ type Manager interface {
 	// RequestDynamicVerificationWithUser verifies a DYNAMIC privilege for a specific user.
 	RequestDynamicVerificationWithUser(privName string, grantable bool, user *auth.UserIdentity) bool
 
+	// VerifyAccountAutoLock Verification Account Auto Lock
```

Comment:
The comment here does not make sense...could you improve it?

Reply:
done